src/uv-common.c

```diff
@@ -113,6 +113,8 @@ const char* uv_err_name(int err) {
   switch (err) {
     UV_ERRNO_MAP(UV_ERR_NAME_GEN)
     default:
+      /* @20170517-sanggyu: To figure out what error number is missing */
+      printf("uv_err_name for (%d) is missing. Add err name string\n", err);
```

Comment:
@LaszloLango Oh, thank you for comment. 
1. Exactly, I thought `fprintf(stderr, ...))`. But it occurred to me that `fprintf(stderr,...)` is not supported in tizenrt-artik05x. (I need to check again.)
2. I totally forgot about TLOG. Thank you remind me. I will use `TDLOG`, and update `TDLOG` for `tizenrt-artik05x` if necessary.
3. 
```
iotjs: /home/travis/build/Samsung/iotjs/deps/libtuv/src/uv-common.c:116: uv_err_name: Assertion `0' failed. 
```
occurs occasionally in travis running. I want to identify the error. It will not affect the binary size after I change to use `TDLOG`.